pom.xml

```
@@ -4,32 +4,37 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.github.json-template</groupId>
+    <groupId>no.ssb.json-template</groupId>
     <artifactId>jsontemplate</artifactId>
-    <version>0.2.1</version>
+    <version>0.3.0</version>
```

I would suggest to change this to `LOCAL-SNAPSHOT` or something similar, since the version now is set a part of github action workflow.
```suggestion
    <version>LOCAL-SNAPSHOT</version>
```